frontend/templates/layouts/mail.html

```diff
@@ -0,0 +1,11 @@
+<html>
+  <head></head>
+  <body>
+    <p>Salaam {{username}},</p>
```

Comment:
we only have the submitter's email so we can't mention their username/name here.

Reply:
Template has been been updated